packages/template/funcs.go

```diff
@@ -874,6 +876,39 @@ func tailTag(par parFunc) string {
 	return ``
 }
 
+func showHideTag(par parFunc, action string) string {
+	setAllAttr(par)
+	cond := par.Node.Attr[`condition`]
+	switch v := cond.(type) {
```

Comment:
I think it's better to do so:
`if v, ok := cond.(string); !ok {
    return ""
}`

Reply:
Fixed